src/view/editor/parts/editor-meeting-rooms.tsx

```diff
@@ -48,15 +74,75 @@ export const EditorMeetingRooms = ({ editorId }: { editorId: string }): ReactEle
 		}
 	}, [meetingRoom, meetingRooms, selection]);
 
-	return meetingRooms ? (
-		<Select
-			items={meetingRooms}
-			background={'gray5'}
-			label={t('label.meeting_room', 'Meeting room')}
-			onChange={onChange}
-			disabled={disabled?.meetingRoom}
-			selection={selection}
-			multiple
-		/>
+	const meetingRoomsAvailability = useMemo(() => {
+		if (meetingRooms?.length > 0) {
```

Comment:
To increase readability and reduce code complexity the condition should be inverted and return earlier